arcusApp/cornea/src/main/java/arcus/cornea/subsystem/cameras/CameraLocalStreamingController.java

```diff
@@ -87,38 +86,14 @@ protected void loadHub() {
                           return;
                       }
 
-                      HubModel hub = hubModels.get(0);
-                      if (!(hub.getCaps().contains(HubSercomm.NAMESPACE))) {
-                          onError(new RuntimeException("Model did not contain proper capabilities; cannot update credentials."));
-                          return;
-                      }
-
-                      username = (String) hub.get(Capability.ATTR_ID);
-                      loadCameraPassword((HubSercomm) hub);
+                      // the below block is meant to run so long as a hub is not a sercomm hub.
```

Comment:
The hubs are from Centralite and Great Star, but there's a sercomm component that connects the cameras to the platform that is not open sourced. The capability model in the platform had this reference removed, but the app still references it.

I think long-term this whole "serccomm" capability is simply going to become video support, but for now we can kill it.